universal-application-tool-0.0.1/app/services/cloud/FileNameFormatter.java

```diff
@@ -0,0 +1,31 @@
+package services.cloud;
+
+import views.applicant.ApplicantProgramBlockEditView;
+
+public class FileNameFormatter {
+
+  /**
+   * Returns the file's original file name, in the format "dev/${filename}" or
+   * "applicant-%d/program-%d/block-%s/${filename}".
+   */
```

Comment:
suggestion (blocking): please expand this comment and clarify why the prefix exists